src/game/object_list_processor.c

```diff
@@ -648,6 +655,8 @@ void update_objects(UNUSED s32 unused) {
 
     // Update all other objects that haven't been updated yet
     update_non_terrain_objects();
+    
+    u32 first = profiler_get_delta(PROFILER_DELTA_COLLISION);
```

Comment:
this could use a comment, seems pretty random. and `first` could be a more specific name as well